findimports.py

```diff
@@ -418,15 +418,23 @@ def warn(self, about, message, *args):
         print(message, file=self._stderr)
         self._warned_about.add(about)
 
-    def parsePathname(self, pathname):
+    def parsePathname(self, pathname, ignores=None):
         """Parse one or more source files.
 
         ``pathname`` may be a file name or a directory name.
+        ``ignores`` is a list of files or directories to ignore.
         """
         if os.path.isdir(pathname):
             for root, dirs, files in os.walk(pathname):
                 dirs.sort()
                 files.sort()
+
+                for ignore in ignores:
```

Comment:
If `ignores` is `None`, this will fail.

Reply:
Since the code itself doesn't allow `None` to be passed, I'd suggest changing the method signature to make `ignores` a positional parameter with no default value.

Reply:
Oh, but I see tests currently don't pass the extra argument.

In which case it may be simplest to 
```suggestion
                for ignore in ignores or []:
```